apps/src/templates/progress/ProgressLesson.jsx

```diff
@@ -39,6 +39,7 @@ const styles = {
   heading: {
     fontSize: 18,
     fontFamily: '"Gotham 5r", sans-serif',
+    cursor: 'pointer'
```

Comment:
Not sure which URL to view one of these, but guessing it needed similar treatment...